ReactSkia/RSkThirdPartyNativeModuleProvider.h

```diff
@@ -14,6 +14,7 @@ extern "C" {
 #endif
 
 RSkLegacyNativeModuleProviderProtocol RSkThirdparyNativeModuleCreatorClassWithName(std::string moduleName);
+xplat::module::CxxModule* RNKeyEventModuleCls(void) RNS_USED; 
```

Comment:
my understanding is these entries will be added by codegen or any other in future. So we do not add these thirdparty package entries